app/src/main/java/org/oppia/android/app/help/faq/faqsingle/FAQSingleActivity.kt

```diff
@@ -18,7 +18,11 @@ class FAQSingleActivity : InjectableAppCompatActivity() {
     (activityComponent as ActivityComponentImpl).inject(this)
     val question = intent.getStringExtra(FAQ_SINGLE_ACTIVITY_QUESTION)
     val answer = intent.getStringExtra(FAQ_SINGLE_ACTIVITY_ANSWER)
-    faqSingleActivityPresenter.handleOnCreate(question, answer)
+    if (question != null) {
+      if (answer != null) {
+        faqSingleActivityPresenter.handleOnCreate(question, answer)
+      }
+    }
```

Comment:
Why are these changes needed?

Reply:
there is some issue in the Gradle build, android studio suggested changing these.